vms/platformvm/state/metadata_delegator.go

```diff
@@ -0,0 +1,16 @@
+// Copyright (C) 2019-2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package state
+
+import "github.com/ava-labs/avalanchego/database"
+
+type delegatorMetadata struct {
```

Comment:
This is currently redundant really, but when dropping stakers we'll extend it to store delegator StartTime.
Introducing the scaffolding here to reduce diffs in main PR